BasicSample/Assets/Shared/Scripts/EnableOnlyDuringRemoting.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+using Microsoft.MixedReality.OpenXR.Remoting;
+
+namespace Microsoft.MixedReality.OpenXR.BasicSample
+{
+    // Enables this GameObject during remoting scenarios and disables it otherwise.
+    // Uses AppRemoting events to observe changes in state even while the GameObject is inactive.
+    public class EnableOnlyDuringRemoting : MonoBehaviour
```

Comment:
Nit on naming:  Maybe better to "EnableOnlyDuringRemotingIsConnected"?   to emphasis the connection.

BTW: shouldn't this script in the "SampleShared/Remoting" folder?

Reply:
good point! Renamed it to "EnableOnlyDuringRemotingConnection"